internal/scan-manager/scanners/common/src/main/java/org/wso2/security/tools/scanmanager/scanners/common/util/FileUtil.java

```diff
@@ -0,0 +1,327 @@
+     */
+    public static void uploadReport(String ftpReportUploadPath, File fileToUpload) throws SftpException,
+            JSchException, FileNotFoundException, ScannerException {
+        ChannelSftp sftp = goToFTPProductLocation(ftpReportUploadPath);
+        uploadFileToFTP(sftp, fileToUpload);
+    }
+
+    /**
+     * Upload the scan report to from container to the FTP location.
+     *
+     * @param sftp         ChannelSFTP to connect t the FTP.
+     * @param fileToUpload scan report file that upload
+     * @throws FileNotFoundException
+     * @throws SftpException
+     */
+    private static void uploadFileToFTP(ChannelSftp sftp, File fileToUpload) throws FileNotFoundException,
+            SftpException, ScannerException {
+            if(fileToUpload != null) {
+                File file = new File(String.valueOf(fileToUpload));
+
+                sftp.put(new FileInputStream(file), file.getName());
+            } else {
+                throw new ScannerException("File that going to upload cannot be null. ");
+            }
+            sftp.disconnect();
+    }
+
+    /**
+     * Create DocumentBuilderFactory with the XXE and XEE prevention measurements.
+     *
+     * @return DocumentBuilderFactory instance
+     */
+    public static DocumentBuilderFactory getSecuredDocumentBuilderFactory() {
+        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+        int entityExpansionLimit = 0;
+        dbf.setNamespaceAware(true);
+        dbf.setXIncludeAware(false);
+        dbf.setExpandEntityReferences(false);
+
+        try {
+            dbf.setFeature(Constants.SAX_FEATURE_PREFIX + Constants.
+                    EXTERNAL_GENERAL_ENTITIES_FEATURE, false);
+            dbf.setFeature(Constants.SAX_FEATURE_PREFIX + Constants.
+                    EXTERNAL_PARAMETER_ENTITIES_FEATURE, false);
+            dbf.setFeature(Constants.XERCES_FEATURE_PREFIX + Constants.
+                    LOAD_EXTERNAL_DTD_FEATURE, false);
+            dbf.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
+        } catch (ParserConfigurationException e) {
+            log.error("Failed to load XML Processor Feature " + Constants.
+                    EXTERNAL_GENERAL_ENTITIES_FEATURE + " or " + Constants.
+                    EXTERNAL_PARAMETER_ENTITIES_FEATURE + " or " + Constants.LOAD_EXTERNAL_DTD_FEATURE +
+                    " or secure-processing.");
+        }
+        SecurityManager securityManager = new SecurityManager();
+        securityManager.setEntityExpansionLimit(entityExpansionLimit);
+        dbf.setAttribute(Constants.XERCES_PROPERTY_PREFIX + Constants.SECURITY_MANAGER_PROPERTY,
+                securityManager);
+        return dbf;
+    }
+
+    /**
+     * Print the pdf report according to the Veracode response result report.
+     *
+     * @param bytesResult Veracode response result
+     * @return whether report is successfully printed
+     * @throws ScannerException
```

Comment:
Let's explain the exceptions in the Util classes. (Applicable across other utils too)